README.md

```diff
@@ -22,8 +22,8 @@ browse the [SMW wiki](https://www.semantic-mediawiki.org) for a more comprehensi
 
 ## Requirements
 
-- PHP 5.3 or later (5.5 or later recommended)
-- MediaWiki 1.19 or later (1.25 or later recommended)
+- PHP 5.5 or later (5.6 or later recommended)
+- MediaWiki 1.23 or later (1.27 or later recommended)
```

Comment:
This is slightly premature, as the latest released version still supports the older stuff, and people look at the stuff on master as docs.